README.md

```diff
@@ -148,6 +149,21 @@ Both `BaseCNN` and `BaseLSTM` also have prediction explanation mechanisms that c
 If you have ideas on how to improve the framework to assess text conversation for constructive and inclusive
 communication, we welcome your contributions!
 
+## Running tests
+### Run tests with a virtual environment
+```python
+python3 -m venv virtualenv-ml-conversational
```

Comment:
Inconsistent use of "python" vs "python3" in the README here

Reply:
python3 is required when creating a virtual env and is no longer needed when running commands in a virtualenv.

There are already steps in the "Installation" on how to create a virtual env and activate it.

Can we move the "running tests" section inside the "Build and Run" section to leverage the installation step so there are no duplicates?

Reply:
I'd argue that python3 is unnecessary as python2 is end-of-life. So depending on the users environment, python could map to python3 anyway.  It depends on how many python versions are installed on your system. MacOS comes with a Python 2.7 as a system default, but other operating systems default with a Py3.x version.  On debian for example, you can define with version python executable points to via update-alternatives.

Reply:
Since we specify in the prerequisite section that the python version needed is `3.6+`, I agree that `python3` is not necessary. I think with the minimum version written, we can safely assume that python will be python3. 

@pramodrj07 could you also help clean up python3 from line 56?

Reply:
I was thinking of changing line 56 is a separate PR. But I can do it in the next change.

Reply:
Done!!

Reply:
The command here does not match the title. This only creates a virtual environment and does not run any tests. Also, I believe you would need an extra step to install requirements.txt as well to be able to run the tests.

I suggested adding "Running test" as part of the "Environment Setup" before. This could help remove the duplicate steps in the test section. WDYT?

Reply:
Yes. I did move it to the 'Environment setup' section